docs/FAQ/gui-faq-en.md

```diff
@@ -4,163 +4,178 @@ Zonemaster
-(registry operator of .fr TLD and several other
-TLDs, e.g. .re, .pm, .tf, .wf, .yt and .paris) and
-[The Swedish Internet Foundation](https://internetstiftelsen.se/en/)
-(registry
-operator of .se and .nu TLDs).
+Zonemaster is a joint project between [AFNIC] (registry of '.fr' TLD and several other
+TLDs, e.g. '.re', '.pm', '.tf', '.wf', '.yt' and '.paris') and the [Swedish Internet Foundation]
+(registry of '.se' and '.nu' TLDs).
 
 <a name="q3"></a>
 #### 3. How can Zonemaster help me?
 The Zonemaster tool is oriented towards two user categories:
 
   - Users who are knowledgable about the DNS protocol.
-  - Users who just want to know whether the the domains owned or used by them
+  - Users who just want to know whether the domain name they own or use
     have any issues or not.
 
 Users of the second category should contact their DNS operator
-as soon as they get the results other than "green" for any
-test of their domain name.
+in case there are errors or warnings for any test of their domain name.
 
 <a name="q4"></a>
-#### 4. Zonemaster returns "Error" or "Warning" for my domain. What does it mean?
-It depends on which test failed for your domain.
+#### 4. Zonemaster returns "Error" or "Warning" for my domain name. What does it mean?
+It depends on which test failed for your domain name.
+Each test are accompanied with one or several messages describing the issues found.
+You can also get further insight about each test from the [Implemented Test Cases] document.
 
 <a name="q5"></a>
-#### 5. How can Zonemaster judge what is right and wrong?
-There is no final judgement of the health of a domain that can be bestowed by
-anyone. The people behind Zonemaster do not claim that the tool is correct in
-every aspect. Sometimes opinions differ. We have done our very best to create a
-default policy for found errors within
-this project. Hopefully this is a good compromise between what is an actual
-potentially dangerous error and what could be merely seen as a notice or warning.
-The added advantage of the tool is that one can add a policy file suited to
-one's necessity to a specified directory and ask the tool to use that policy
-file when running the tests.
-But as with all things as evolving as DNS the situation is most likely
-changing, what is a notice today could be an error tomorrow. If you really think
-we have made a mistake in our judgement please do not hesitate to drop us an email
-at [zonemaster-users@lists.iis.se](mailto:zonemaster-users@lists.iis.se) (moderated
-mailing list) with a link to your test and an explanation why you think it
-shows something that you consider incorrect.
+#### 5. How can Zonemaster distinguish between what is right and wrong?
+The judgement of Zonemaster is primarily based on the DNS standards as defined in [RFCs].
+It also bases its judgement on DNS best practices, which can be more loosely defined.
+All Zonemaster tests are defined in [Test Case Specifications][Implemented Test Cases]
```

Comment:
It is more helpful to refer to this is a link to all test cases with the test case description, `[Defined Test Cases]`.

Reply:
Updated